source/javascripts/services/_appService.js.erb

```diff
@@ -38,6 +38,10 @@
 				betaTags: undefined
 			};
 
+			if (requestService.mode === "cli") {
+				appService.pipelineConfig = {}
+			}
+
```

Comment:
This implies that pipeline config makes sense in CLI environment, whereas it doesn't.

I think it would be better to wherever `appService.pipelineConfig` is used, prefix it with checking whether we're in website mode or not.

This current solution works for simple properties like `.usesRepositoryYml`, but would break, if we were to introduce a nested property & do checks for it, e. g.

`if (appService.pipelineConfig.repoSettings.provider)`